plugins/source/crowdstrike/codegen/recipes/alerts.go

```diff
@@ -0,0 +1,17 @@
+package recipes
+
+import (
+	"github.com/crowdstrike/gofalcon/falcon/models"
+)
+
+func Alerts() []*Resource {
+	resources := []*Resource{
+		{
+			Service:    "alerts",
+			SubService: "query",
+			Struct:     &models.MsaQueryResponse{},
```

Comment:
It needs a PK field... Otherwise there is no way to do an update/upsert

Reply:
the api itself doesnt' ahve a PK field. Not sure what is the recommendation here.

Reply:
instead of using the `MsaQueryResponse` struct I think we should actually be storing the contents of 
`models.MsaQueryResponse.resources` which is of type `DomainEnvironmentScore` ...
We don't need the `errors` or `meta` from the request stored in the destination... This way we can also use the `id` column as a PK

Reply:
I am not sure if i am looking at the right place, but I don't see DomainEnvironmentScore as a type here https://github.com/CrowdStrike/gofalcon/blob/main/falcon/models/msa_query_response.go#L33

Reply:
I mistakenly put this comment in the wrong place it should be in `plugins/source/crowdstrike/codegen/recipes/incidents.go`

Reply:
For this resource do we need to be storing the `errors` and `meta` or could we have a single `text` column and put a PK on it?

Reply:
For the alert fatigue usecase, we don't need errors and meta. Lemme know what you think, but i think putting a PK on the text field doesn't feel intuitive, it might be same value as cq_id.